bin/typescript-redux-query-petstore-with-npm-version.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+SCRIPT="$0"
```

Comment:
please add a windows version of this script in `bin/windows`

Reply:
Done and pushed. Haven't tested them yet, I don't have easy access to a windows machine/VM currently.